engines/pyfunc-ensembler-job/requirements.txt

```diff
@@ -4,9 +4,9 @@ jinjasql==0.1.8
 jinja2==3.0.3
 mlflow==1.14.1
 numpy==1.21.6
-pandas==1.2.2
+pandas==1.3.5
 py4j==0.10.9
-pyarrow==3.0.0
+pyarrow==6.0.1
 pyspark==3.0.1
-pyyaml==5.4.1
+pyyaml==6.0
```

Comment:
Upgraded this to the earliest minor version (with the latest patch version) that supports Python 3.7 to 3.10: https://pypi.org/project/PyYAML/6.0/#files